### DISCUSSION TITLE
Enhanced songs in your liked playlist wont play

### DISCUSSION THREAD

**Discussion by ghost** [SpotX-Official/SpotX#49] [Errors]
Spotify Version: 1.1.86.857.g3751ee08
Plan Spotify: Free
Windows Version: Windows 10 21H2, OS build 19044.1288
PowerShell Version:  5.1.19041.1237
1. Clicking the enhance playlist button correctly adds multiple new songs to your liked songs playlist.
2. However, When attempting to play the song your met with a blank bar, play button being grayed out
3. The added songs will be skipped by the queue automatically
![WontPlay](https://user-images.githubusercontent.com/45320697/170803903-99495ac4-0930-4391-8d3a-ca8f6a74bd55.PNG)

**Comment by amd64fox** [comment]
Hello, thanks for the feedback, this is not a bug of the mod, this is a new experimental feature, Spotify only added it, but did not activate it due to the fact that it has not yet been finalized.
It may take a few days or weeks for `1.1.86.857` to work, or you may need to wait for a new version of the client.

[Temporary solution](https://github.com/amd64fox/SpotX/discussions/47) for you to disable this experimental feature. looking for this `Enable Enhance Liked Songs UI and functionality`

**Reply by DimaSheiko** [reply]
When did Spotify introduce this feature? Too bad it's not available in your liked playlist 😔

**Reply by amd64fox** [reply]
on the desktop it appeared a few days ago with the release of `1.1.86`, in the mobile application they have been testing selectively for a month now.

**Comment by amd64fox** [comment]
Temporarily disabled this feature until the developers fix it.

**Reply by DimaSheiko** [reply]
I like it, and it just doesn't work in the favorite sounds playlist. In the normal playlist it works fine.

**Reply by amd64fox** [reply]
Temporary disable only applies to enhance liked songs

**Reply by amd64fox** [reply]
Fixed in latest update

**Reply by DimaSheiko** [reply]
OMG! I will try!

**Reply by DimaSheiko** [reply]
It didn't work for me. I have the version `1.1.89.858.ge2edefbd` When I hit next, and the next track is an enhanced song, it jumps to the next one and not to the recommended one.

**Reply by amd64fox** [reply]
clean install

**Reply by DimaSheiko** [reply]
It does not work for me, I have already uninstalled and installed it 3 times. I have used [this](https://github.com/amd64fox/Uninstall-Spotify) and [this](https://cutt.ly/dErpPEK) and it still doesn't work

**Reply by amd64fox** [reply]
After a clean install, check your version again here

![2](https://user-images.githubusercontent.com/62529699/177645261-3693e7b7-42d2-42e3-99e7-dd2966c534e4.jpg)

**Reply by mreduar** [reply]
It doesn't work for me either
![image](https://user-images.githubusercontent.com/22550119/177645837-d8ac2d15-a46d-4b58-82d1-445d9a73f3e0.png)
![powershell_DsxaJNkHUv](https://user-images.githubusercontent.com/22550119/177645849-640cd023-daf7-4df1-9954-43a5b2c5f76e.gif)

**Reply by DimaSheiko** [reply]
I have version `1.1.89.858.ge2edefbd`, I even uninstalled it with Revo Uninstaller Pro Program.

**Reply by amd64fox** [reply]
Strange... it works for me
![6](https://user-images.githubusercontent.com/62529699/177649181-df451d1c-1b0d-4f3b-9291-94a6899d7d00.gif)
okay, I’ll look a little more later, maybe it’s related to different types of accounts

**Reply by amd64fox** [reply]
There was a migration to a new version, try a clean install again.
If it doesn’t work again, then most likely it is connected with different types of accounts, you need to wait for the next update.

**Reply by DimaSheiko** [reply]
Still not working 😢

**Reply by DimaSheiko** [reply]
I have installed the latest version with its recent changes and now this feature works! Thank you very much! ❤

**Reply by amd64fox** [reply]
it's all the same version `1.1.89.862.g94554d24`
Tell us more about all your actions after which the function worked.

**Reply by DimaSheiko** [reply]
What I did was instead of completely uninstalling the app I didn't uninstall it I ran the script and overwrote the previous app. But the rest I didn't do anything else. 
I didn't really test the functionality before installing your latest changes to check if it was a cache problem last time

**Reply by amd64fox** [reply]
Weird, but let's hope it doesn't happen again.

**Reply by luinbytes** [reply]
> Weird, but let's hope it doesn't happen again.

I hate necroing old threads, but I'm having this exact issue on a premium account.

**Reply by amd64fox** [reply]
> I'm having this exact issue on a premium account

You won't believe it, Spotify developers are still testing these features for the desktop version, I encountered a similar error about two weeks ago and it went away by itself, most likely these are problems with the server side of the function, but you can try to perform a [clean installation](https://telegra.ph/SpotX-FAQ-09-19#1.-Clean-installation), it will not hurt .
